### DISCUSSION TITLE
Migration path for native Link fields

### DISCUSSION THREAD

**Discussion by gateszies** [craftcms/cms#15695] [Ideas]
We're excited to use the native Link fields now that they've been expanded in Craft 5.3. Most of our links are [Hyper](https://verbb.io/craft-plugins/hyper/features) fields and we'd like to migrate to using the native Link field. It'd be great to have a path for migration from Hyper to Link fields similar to what Hyper uses to migrate from Typed Link to Hyper.

**Comment by davidwebca** [comment]
Same, I've had some niche and complicated issues with Hyper lately and I'd rather work with the native one as soon as possible.
